repology/parsers/parsers/openvsx.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2024 Gavin John <gavinnjohn@gmail.com>
+#
+# This file is part of repology
+#
+# repology is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# repology is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with repology.  If not, see <http://www.gnu.org/licenses/>.
+
+import json
+
+from typing import Iterable
+
+from repology.package import LinkType
+from repology.packagemaker import NameType, PackageFactory, PackageMaker
+from repology.parsers import Parser
+
+
+class OpenVSXParser(Parser):
+    def iter_parse(self, path: str, factory: PackageFactory) -> Iterable[PackageMaker]:
+        with open(path, 'r') as extdatafile:
+            extension_data = json.load(extdatafile)
```

Comment:
Loading json into memory is not scalable. You should use `iter_json_list` to parse it as stream.